### PR TITLE
[RSDK-2002] Move slam config & tests to slam repo

### PR DESCRIPTION
**Pull request by kkufieta**
Ticket: https://viam.atlassian.net/browse/RSDK-2002

* Move the slam/internal/config code & related tests to the slam repo
* Rename testhelper file
* Import config code back into builtin

### CODE REVIEW

**Review by zaporter-work — COMMENTED** [review]
Should this PR also change the slamConfig in the other files like cartographer_int_test.go to use the new config in go.viam.com/slam?

**Comment by kkufieta** [comment]
@zaporter-work The slamConfig is not used in cartographer_int_test.go (as we found out in our offline discussion)

**Review by tessavitabile — APPROVED** [review]
It feels a little risky to remove these tests now before we've set up go CI for the slam repo, but since the config code is pretty stable, I'm okay with the risk.

File: web/runtime-shared/static/main.css [hunk 1]
```diff
@@ -1 +1 @@
...-.72 2.64-.03 3.85.68 1.18 1.89 1.89 3.24 1.89h40.51c1.35 0 2.56-.71 3.23-1.89.7-1.21.69-2.65-.02-3.85zm-25.53-21.405h3.381v3.187l-.724 8.92H24.66l-.725-8.92v-3.187zm2.97 17.344a1.712 1.712 0 01-1.267.543c-.491 0-.914-.181-1.268-.543a1.788 1.788 0 01-.531-1.297c0-.502.176-.935.53-1.297a1.712 1.712 0 011.269-.544c.49 0 .914.181 1.268.544s.53.795.53 1.297c0 .503-.176.934-.53 1.297z'/%3e %3c/svg%3e") no-repeat}.placeholder\:text-gray-400::placeholder{--tw-text-opacity: 1;color:rgb(156 163 175 / var(--tw-text-opacity))}.focus\:outline-none:focus{outline:2px solid transparent;outline-offset:2px}@media (min-width: 640px){.sm\:flex-nowrap{flex-wrap:nowrap}.sm\:border-l{border-left-width:1px}}:root{--popper-theme-background-color: #fff;--popper-theme-background-color-hover: #fff;--popper-theme-text-color: #000;--popper-theme-border-color: #000;--popper-theme-border-width: 1px;--popper-theme-border-style: solid;--popper-theme-border-radius: 0px;--popper-theme-padding: 4px;--popper-theme-box-shadow: 0 6px 30px -6px rgba(0, 0, 0, .25)}.subtitle[data-v-03745d95],.subtitle[data-v-dbe09de2]{color:var(--black-70)}#source{position:relative;width:50%;height:50%}h3{margin:.1em;margin-block-end:.1em}
```
Comment (nicksanford) [inline web/runtime-shared/static/main.css:1]: Why is this file changing?

**Review by nicksanford — CHANGES_REQUESTED** [review]

**Review by zaporter-work — APPROVED** [review]

File: web/runtime-shared/static/main.css (hunk 1 above)
Comment (kkufieta) [reply web/runtime-shared/static/main.css]: Fixed, it's not anymore.

**Review by nicksanford — APPROVED** [review]

**Pull request merged**